### ISSUE TITLE
Autolayout not working 

### ISSUE THREAD

**Issue by yawboafo** [chenyun122/StepIndicator#17]
Applying constrains programatically to this seems not to work

**Comment by korantengNanaYaw** [comment]
WidthAnchor and HeightAnchor somehow does not affect the view , you will have to set the view width and height using the Frame .

**Comment by derlysbenito** [comment]
I face the same autolayout problem. Could it be solved? @yawboafo @korantengNanaYaw

**Comment by chenyun122** [comment]
Thanks for using, I am checking the problem.

**Comment by chenyun122** [comment]
I added an example for applying constraints programmatically. It's working without the problems mentioned. Please check the code in file `ViewController.swift`, or bellow code:

``` swift
// Example for applying constrains programmatically
func applyNewConstrains() {
   // ......

   // Add new constrains programmatically
   self.stepIndicatorView.widthAnchor.constraint(equalToConstant: 263.0).isActive = true
   self.stepIndicatorView.heightAnchor.constraint(equalToConstant: 80.0).isActive = true
   self.stepIndicatorView.centerXAnchor.constraint(equalTo: self.view.centerXAnchor).isActive = true
   self.stepIndicatorView.topAnchor.constraint(equalTo: self.view.layoutMarginsGuide.topAnchor, constant:30.0).isActive = true
    
   self.scrollView.topAnchor.constraint(equalTo: self.stepIndicatorView.bottomAnchor, constant: 8.0).isActive = true
}
```

The running effect with constrains added programmatically:
![constraints](https://user-images.githubusercontent.com/3788227/63499622-c6f00e80-c4fa-11e9-8793-40dce617eb4a.png)

**Comment by yawboafo** [comment]
@chenyun122 
Try creating the step Indicator programatically , remove and delete the reference from story board., to reproduce the issue i'm facing

``` swift
lazy var stepIndicatorView: StepIndicatorView = {
		let view = StepIndicatorView()
		view.numberOfSteps  = 3
		view.circleStrokeWidth = 1.5
		view.circleRadius = 14
		view.lineMargin = 4.0
		view.lineStrokeWidth = 2.0
		view.displayNumbers = true
		view.direction = .leftToRight
		view.translatesAutoresizingMaskIntoConstraints = false
		return view
	}()
```

The fix i used was that i had to define 
```swift
 view.frame.size.width = 300
 view.frame.size.height = 100`
```
i still could not set widthAnchor without defining the frame size

**Comment by chenyun122** [comment]
> @chenyun122
> Try creating the step Indicator programatically , remove and delete the reference from story board., to reproduce the issue i'm facing
> 
> ```swift
> lazy var stepIndicatorView: StepIndicatorView = {
> 		let view = StepIndicatorView()
> 		view.numberOfSteps  = 3
> 		view.circleStrokeWidth = 1.5
> 		view.circleRadius = 14
> 		view.lineMargin = 4.0
> 		view.lineStrokeWidth = 2.0
> 		view.displayNumbers = true
> 		view.direction = .leftToRight
> 		view.translatesAutoresizingMaskIntoConstraints = false
> 		return view
> 	}()
> ```
> 
> The fix i used was that i had to define
> 
> ```swift
>  view.frame.size.width = 300
>  view.frame.size.height = 100`
> ```
> 
> i still could not set widthAnchor without defining the frame size

@yawboafo , thanks for the helpful details. The bug is now fixed with the new version 1.0.8. Please update the lib with pod or source code.

**Comment by chenyun122** [comment]
@yawboafo , thanks for the helpful details. The bug is now FIXED with the new version 1.0.8. Please update the lib with pod or source code.

**Comment by yawboafo** [comment]
Thanks it works now !
